addons/mail/static/tests/translation/translation_test.js

```diff
@@ -0,0 +1,50 @@
+/* @odoo-module */
+
+import { click, start, startServer, contains } from "@mail/../tests/helpers/test_utils";
+
+const spanishText = "Al mal tiempo, buena cara.";
+const englishText = "To bad weather, good face.";
```

Comment:
These consts are useless indirection. Prefer hard-code in test, so data are explicit

Reply:
Removed.